hack/update-codegen.sh

```diff
@@ -20,11 +20,6 @@ set -o pipefail
 
 source $(dirname $0)/../vendor/github.com/tektoncd/plumbing/scripts/library.sh
 
-SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
```

Comment:
I hope this one wasn't slowing us down :P

Reply:
This one wasn't, but ain't used anymore :stuck_out_tongue_closed_eyes: